shared/config/i18n.ts

```diff
@@ -101,7 +106,8 @@ export const LangCodes = {
   [Languages.Japanese]: 'ja',
   [Languages.German]: 'de',
   [Languages.Arabic]: 'ar',
-  [Languages.Swahili]: 'sw'
+  [Languages.Swahili]: 'sw',
+  [Languages.Korean]: 'ko'
 };
 
 /**
```

Comment:
Down on line 116, add `Languages.Korean` to the `hiddenLangs` array.

That'll make this PR safe to merge without blocking deployments. We'll unhide the language once the Korean site itself is live.

Reply:
done 👍🏼 thanks @naomi-lgbt 😄